utils.groovy

```diff
@@ -155,17 +155,6 @@ def triggered_by_push() {
     return (currentBuild.getBuildCauses('com.cloudbees.jenkins.GitHubPushCause').size() > 0)
 }
 
-// Runs closure if credentials exist, otherwise gracefully return.
-def tryWithCredentials(creds, Closure body) {
```

Comment:
are the uses of `tryWithCredentials` in this file safe after this removal?

Reply:
Yup, it's similar to our use of `shwrap` in the file.